.cspell.json

```diff
@@ -34,9 +36,7 @@
     "Typeorm",
     "typescript",
     "ukef",
-    "venv",
-    "probabilityof",
```

Comment:
Did we deliberately delete `"probabilityof"` here?

Reply:
`"probabilityof"` is duplicated further up, I was just tidying the file